src/Beberlei/Metrics/Collector/CRedis.php

```diff
@@ -0,0 +1,66 @@
+<?php
+/**
+ * Beberlei Metrics.
+ *
+ * LICENSE
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this package in the file LICENSE.txt.
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to kontakt@beberlei.de so I can send you a copy immediately.
+ */
+namespace Beberlei\Metrics\Collector;
+
+use Credis_Client;
+
+class CRedis implements Collector
+{
+    /** @var \Credis_Client */
+    private $credis_client;
+
+    public function __construct(Credis_Client $credis_client)
+    {
+        $this->credis_client = $credis_client;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function increment($variable)
+    {
+        $this->credis_client->incr($variable);
```

Comment:
https://redis.io/commands/incr/

```
Note: this is a string operation because Redis does not have a dedicated integer type. The string stored at the key is interpreted as a base-10 64 bit signed integer to execute the operation.
```

Once you reach the limit of 2 pow 63 + 1, you will get an error

https://stackoverflow.com/questions/36861472/what-happens-when-int64-maxvalue-is-exceded-with-redis-incr